tauri-app/src/__tests__/pickConfig.test.ts

```
@@ -1,127 +1,196 @@
+          network: {
+            key: 'network1',
+            rpc: 'rpc-url',
+            chainId: 14,
+          },
         },
       },
     },
   },
-  charts: {},
-  deployments: {
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)